contrib/README.md

```diff
@@ -0,0 +1,70 @@
+# How to start lorri automatically with systemd
+
+This guide shows you how to set up a systemd socket and service file that start
+`lorri daemon` automatically on demand.
+
+## Using systemd directly
+
+If your `lorri` binary is not in `~/.nix-profile/bin/lorri`, please change the
+`ExecStart=` setting in `lorri.service` to the correct location.
+
+We will use a systemd _user_ socket and service, so let's create the relevant
+directory if it does not already exist, and then copy over the socket and
+service definitions:
+
+```bash
+$ mkdir -p ~/.config/systemd/user
+$ cp contrib/lorri.{socket,service} ~/.config/systemd/user/
+```
+
+At this point, reload the systemd user daemon. The socket and the service files
+should then be visible to systemd:
+
+```bash
+$ systemctl --user daemon-reload
+$ systemctl --user list-unit-files | grep lorri
+lorri.service                         static  
+lorri.socket                          disabled
+```
+
+Now start the socket:
+
+```bash
+$ systemctl --user start lorri.socket
```

Comment:
Use `systemctl --user enable --now lorri.socket` instead?

Reply:
No: this socket is not mean to be `enable`d. We can start it here; on system reboot it will be started again because it is `WantedBy=sockets.target`.